src/main/java/org/mskcc/cbio/oncokb/config/cache/CachingConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package org.mskcc.cbio.oncokb.config.cache;
+
+import org.springframework.cache.annotation.CachingConfigurerSupport;
+import org.springframework.cache.interceptor.CacheErrorHandler;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class CachingConfiguration extends CachingConfigurerSupport {
+    @Override
+    public CacheErrorHandler errorHandler() {
+        return new LoggingCacheErrorHandler();
+    }
+}
+
```

Comment:
I'm wondering if its the same if we extend CacheConfiguration (`src/main/java/org/mskcc/cbio/oncokb/config/CacheConfiguration.java`) with `CachingConfigurerSupport` and then register the errorHandler in the same file?

Reply:
That's a good point. Let me try.

Reply:
@calvinlu3 updated